tests/vcs/git/GitCommitBearTest.py

```diff
@@ -294,7 +294,8 @@ def test_check_issue_reference(self):
                         'First line, blablablablablabla.\n'
                         'Another line, blablablablablabla.\n'
                         'Closes #01112')
-        self.assertEqual(self.run_uut(body_close_issue=True), [])
+        self.assertEqual(self.run_uut(body_close_issue=True),
+                         ['No host configured in path.'])
```

Comment:
why this change?

Reply:
This was done to check for some return value in host :sweat_smile: 
Will revert it